Zero-Collateral/CeFi-Frontend
# Blocknative integration

## Description

Adds BlockNative Wallet integration.

## Testing
1. Start the app using `npm run start:dev`
2. Start Metamask and switch to the Ropsten network.
3. In the top right, click `Connect`. You should see a popup to connect Web3. Click Connect Again.
4. The BlockNative popup should appear and MetaMask should be an option. Click it.
5. The app should JustWork.com.
